src/pages/Cart/index.js

```diff
@@ -0,0 +1,60 @@
+import { ProductRow } from "../../components";
+import ErrorBoundary from "../../error";
+import { useCart } from "../../hooks";
+import { Button, Flex, StyledLink, FullWidthTable } from "../../styled";
+import { formatter } from "../../utils";
+
+function Cart() {
+  const { products, total, dispatch, reset } = useCart();
```

Comment:
Same here with the reset, if you import `useCart` in the same component as `useInput` and you're required to clean both, you'll have a conflict because both of them are named `reset`